tests/Temporalio.Tests/Worker/WorkflowWorkerTests.cs

```diff
@@ -7395,6 +7396,95 @@ await ExecuteWorkerAsync<WorkflowUsingPriorities>(
             new TemporalWorkerOptions().AddAllActivities<WorkflowUsingPriorities>(null));
     }
 
+    [Workflow]
+    public class WaitOnSignalWorkflowThenActivity
```

Comment:
```suggestion
    public class WaitOnSignalWorkflowThenActivityWorkflow
```

Pedantic but suffixing with `Workflow` is common